frontend/pages/club/[club]/edit.js

```diff
@@ -262,6 +272,39 @@ class ClubForm extends Component {
     })
   }
 
+  submitEvent(data) {
+    doApiRequest(`/clubs/${this.state.club.code}/events/?format=json`, {
+      method: 'POST',
+      body: data
+    }).then(resp => {
+      if (resp.ok) {
+        this.notify(`Event has been created!`)
+        this.componentDidMount()
+      } else {
+        resp.json().then(err => {
+          this.notify(formatResponse(err))
+        })
+      }
+    })
+  }
+
+  deleteEvent(id) {
+    console.log(this.state.club.code)
+    console.log(`/clubs/${this.state.club.code}/events/${id}`)
```

Comment:
Remove all of these `console.log`s when you're ready to merge! Run `npm run lint --fix` to check for style issues similar to this one.